backend/src/main/java/com/hooria/todo/domain/ActivityLog.java

```diff
@@ -2,18 +2,27 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.ToString;
 
 import java.time.LocalDateTime;
 
 @Getter
+@ToString
 @AllArgsConstructor
```

Comment:
정적 팩토리 메서드로 생성자를 대신한다면 아래처럼 외부에서 접근못하도록 하는 것도 추천드립니다.

```suggestion
@AllArgsConstructor(access = AccessLevel.PRIVATE)

```

Reply:
오오오 좋은 팁 감사드립니다!! 🤩🤩🤩